pm/utils/NLPERMANENTMARKERSHOSTSParserCallbacks.m

```diff
@@ -33,13 +44,24 @@ void NLPERMANENTMARKERSHOSTSonHostEntryFound(NLPERMANENTMARKERSHOSTS_CHostEntry
 
 // error handler for the hosts file parser
 void NLPERMANENTMARKERSHOSTSonHostsParseError(NLPERMANENTMARKERSHOSTS_CHostEntryError * error) {
-    NSString *windowTitle = NSLocalizedString(@"Parse error", @"error: title");
-    NSString *errorFormatString = NSLocalizedString(@"In /private/etc/hosts on line:%i %s\n\n%s\n\nShould this line be permanently removed from /private/etc/hosts?\nChoose no to quit Hosts and manually correct the error.",
-                                                             @"error: description");
+    LocalizationSupportClass *foo = [[LocalizationSupportClass alloc] init];
+    NSBundle *myBundle = [NSBundle bundleForClass:[foo class]];
+
+    NSString *windowTitle = [myBundle localizedStringForKey:@"Parse error" value:@"" table:nil];
```

Comment:
Why not use NSLocalizedStringUniversal here?

Reply:
Because NLPERMANENTMARKERSHOSTSParserCallbacks is not a class I can't use [self class]. Please see my additions for LocalizationSupportClass a few lines up.